lib/qesapdeployment.pm

```diff
@@ -438,6 +439,19 @@ sub qesap_ansible_log_find_timeout
     return $timeout_match ? 1 : 0;
 }
 
+=head3 qesap_ansible_log_find_missing_sudo_password
+
+    Return the '"msg": "Missing sudo password"' error found in the Ansible log or not
+=cut
+
+sub qesap_ansible_log_find_missing_sudo_password
```

Comment:
This one is mostly the same of `qesap_ansible_log_find_timeout` : only difference is the string to match. I see room of improvement to merge the two function and allow the caller to specify the string to mach as external argument.

Reply:
Yes, you are right. We can open a new ticket for this.